docs/application/native/guides/app-management/app-controls.md

```diff
@@ -664,6 +669,126 @@ create_base_gui(appdata_s *ad)
    }
    ```
 
+<a name="action"></a>
+## Handle Application Control Actions
+
+You can define and handle each application control actions.
+
+- Defining the application control action
+
+    > **Note**
```

Comment:
@mijin85cho Basically we keep 'Note' form to ensure user checks it so that point should not be left. For additional information or tips we use brackets to highlight them or tips-tag (not used in Tizen so far). This point feels more like a note as we usually keep 'Since Tizen 2.4...' in note form. In my opinion it should have some description first then this note, Even if you still feel to change it to normal form, kindly let me know I'll request regarding the same.